assets/reducers.js

```diff
@@ -133,8 +133,10 @@ export function defaultReducer(state, action) {
         return {...state, query: action.query, activeItem: null, search: search};
     }
 
-    case QUERY_ITEMS:
-        return {...state, isLoading: true, totalItems: null, activeQuery: state.query};
+    case QUERY_ITEMS: {
+        const resultsFiltered = !isEmpty(get(state, 'search.activeFilter')) || !isEmpty(get(state, 'search.createdFilter'));
```

Comment:
I'm not sure if we should keep on state something derived from state, that's kind of breaking this single source of truth idea, we should probably have selector defined for it and reuse that if we need it in more places

Reply:
I think we need to understand if a query took place. User can set a filter or change the date range, so `search.activeFilter` or `search.createdFilter` assigned some values. But that doesn't mean the results are filtered yet until they click the `search` button. That's why I moved this line from selector to the action.